src/govuk/components/modal-dialogue/template.njk

```diff
@@ -0,0 +1,69 @@
+{% from "../button/macro.njk" import govukButton %}
+{% set headingLevel = params.headingLevel if params.headingLevel else 2 %}
+{% set labelledBy = params.labelledBy if params.labelledBy else params.id + "-title" %}
+{% set describedBy = params.describedBy if params.describedBy else params.id + "-description" %}
+<div id="{{ params.id }}" class="govuk-modal-dialogue{% if params.classes %} {{ params.classes }}{% endif %}{% if params.open %} govuk-modal-dialogue--open{% endif %}" data-module="govuk-modal-dialogue">
+  <div class="govuk-modal-dialogue__wrapper">
+    <dialog class="govuk-modal-dialogue__box" aria-labelledby="{{ labelledBy }}" aria-describedby="{{ describedBy }}" aria-modal="true" role="{{ params.role | default('dialog') }}" tabindex="0"
+    {%- for attribute, value in params.attributes %} {{ attribute }}{% if value %}="{{ value }}"{% endif %}{% endfor %}{% if params.open %} open{% endif %}>
+      <div class="govuk-modal-dialogue__header">
+        {#- We use an inline SVG for the crown so that we can cascade the
+        currentColor into the crown whilst continuing to support older browsers
+        which do not support external SVGs without a Javascript polyfill. This
+        adds approximately 1kb to every page load.
+        We use currentColour so that we can easily invert it when printing and
+        when the focus state is applied. This also benefits users who override
+        colours in their browser as they will still see the crown.
+        The SVG needs `focusable="false"` so that Internet Explorer does not
+        treat it as an interactive element - without this it will be
+        'focusable' when using the keyboard to navigate. #}
+        <svg
+          role="presentation"
+          focusable="false"
+          class="govuk-modal-dialogue__crown"
+          xmlns="http://www.w3.org/2000/svg"
+          viewbox="0 0 132 97"
+          height="30"
+          width="36"
+        >
```

Comment:
Had a chat with @FeyAgape. We should consider removing the crown logo as there is no clear user need. Its usage here is not consistent with our other components and it adds complexity. 

Its usage in public facing sites in this manner might also be problematic with regards to the branding guidelines; the [Service Manual guidance](https://www.gov.uk/service-manual/design/making-your-service-look-like-govuk#if-your-service-isnt-on-govuk) only mentions the use of the crown logo in the header.